src/dataServices/apiDataService.ts

```diff
@@ -0,0 +1,123 @@
+/* eslint-disable class-methods-use-this */
+
+import {
+    ConditionalDeleteResourceRequest,
+    CreateResourceRequest,
+    DeleteResourceRequest,
+    GenericResponse,
+    PatchResourceRequest,
+    Persistence,
+    ReadResourceRequest,
+    ResourceNotFoundError,
+    UpdateResourceRequest,
+    vReadResourceRequest,
+} from 'fhir-works-on-aws-interface';
+import axios from 'axios';
+
+export class ApiDataService implements Persistence {
+    updateCreateSupported: boolean = false;
+
+    // TODO: Pull this value from AWS Param Store
+    static readonly INTEGRATION_TRANSFORM_URL = 'http://localhost:4000';
+
+    async createResource(request: CreateResourceRequest): Promise<GenericResponse> {
+        let genericResponse: GenericResponse = { message: '', resource: {} };
```

Comment:
Refactor to use `const` instead of `let`. I don't think mutation is warranted here

Also, is `{ message: '', resource: {} }` ever a valid response?

Reply:
Initially I used a constant for `genericResponse`. However, if I don't return a `genericResponse `at the end of the method eslint will give me the error `Function lacks ending return statement and return type does not include 'undefined'.`

It was because my `handleError` function was throwing an error (as expected) but eslint didn't know that. I think I can refactor this so `handleError` becomes `getError`. Then I can throw that error and eslint will be happy.